lib/screens/dup_screen_data/request.ex

```diff
@@ -37,6 +37,7 @@ defmodule Screens.DupScreenData.Request do
     "Needham Heights" => "Needham Hts",
     "Houghs Neck via McGrath & Germantown" => "Houghs Neck via McGth & Gtwn",
     "Houghs Neck via Germantown" => "Houghs Neck via Germntwn",
+    "Middleborough/Lakeville" => "Middleborough / Lakeville",
```

Comment:
Is there a possibility of any other format?

Reply:
I searched around in the API and that's the only format I found for that headsign.

Reply:
oh sweet! thanks for the extra mile check